packages/core/src/api/bot.ts

```diff
@@ -0,0 +1,91 @@
+import { makeURLSearchParams, type REST } from '@discordjs/rest';
+import {
+	Routes,
+	type APIGuildMember,
+	type APIDMChannel,
+	type APIPartialGuild,
+	type APIUser,
+	type RESTGetAPICurrentUserGuildsQuery,
+	type RESTPatchAPICurrentUserJSONBody,
+	type RESTPatchAPIGuildMemberJSONBody,
+	type RESTPatchAPIGuildVoiceStateCurrentMemberJSONBody,
+} from 'discord-api-types/v10';
+
+export class BotsAPI {
+	public constructor(private readonly rest: REST) {}
+
+	/**
+	 * Returns the user object of the requester's account
+	 */
+	public async getUser() {
```

Comment:
```suggestion
	public getUser() {
```
@almeidx you forgot to remove `async`

Reply:
it was done to make eslint happy 😄